Tools/scripts/build_docs.sh

```diff
@@ -5,11 +5,12 @@ set -e
 # work from either APM directory or above
 [ -d ArduPlane ] || cd APM
 
-export DOCS_OUTPUT_BASE=../buildlogs/docs
```

Comment:
This seems a functional-ish change... what's the rationale?

Reply:
1- buildlogs will be out of ardupilot repository.
2- This folder does not exist by default.
3- It's not possible to redirects output to a file if the parent folder does not exist.

Reply:
+1 to the change. We do this for a number of the scripts, which makes sense on the autotest server, but on my local machine just pollutes other folders randomly (I actually had a conflict with a parent folder like this before). As a general sanity thing I think we should always be keeping or contents in the root ArduPilot folder and below. (Unless a script is an auto test server only, but even then that should come with caveats about what it will do)